docs/features/kustomize/rollout_cr_schema.json

```diff
@@ -47,6 +47,22 @@
                                                     "type": "array",
                                                     "x-kubernetes-patch-merge-key": "name",
                                                     "x-kubernetes-patch-strategy": "merge"
+                                                },
```

Comment:
why is dry run showing in teh diff?

Reply:
I guess previously the `codegen` for dryRun was skipped